documentation/adoc/appendix_metrics.adoc

```diff
@@ -25,10 +25,10 @@ where `[namespace]` is the namespace/project where the Apache Kafka cluster was
 
 After that, it's possible to deploy the Prometheus server in the cluster :
 
-. Create the https://github.com/strimzi/strimzi/blob/master/metrics/examples/prometheus/openshift-template.yaml[provided "prometheus" template] by running
+. Create the https://github.com/strimzi/strimzi/blob/master/metrics/examples/prometheus/kubernetes.yaml[provided "prometheus" template] by running
```

Comment:
Why using the full URL and not just `metrics/examples/prometheus/kubernetes.yaml` ?